backend/tracim_backend/lib/core/event.py

```diff
@@ -867,6 +866,9 @@ def _has_just_been_undeleted(self, obj: Union[User, Workspace, ContentRevisionRO
 
 
 def get_event_user_id(session: TracimSession, event: Event) -> typing.Optional[int]:
+    # INFO - G.M - 2022-01-10 - user is None case
```

Comment:
Could you add in which circumstances (old Tracim version…) it could happen?

Reply:
```suggestion
    # INFO - G.M - 2022-01-10 - user is None case,
    # may happened with user deletion through tracimcli before tracim 3.9.
```

Reply:
@grignards can you validate this proposal ?